docs/user/openstack/kuryr.md

```diff
@@ -37,6 +40,8 @@ As highlighted in the minimum quota recommendations, when using Kuryr SDN, there
 openstack quota set --secgroups 100 --secgroup-rules 500 --ports 500 --subnets 100 --networks 100 <project>
 ```
 
+**NOTE:** Each Amphora Load Balancer creates a VM. Even if that VM is not part of the user quota, the OpenStack cluster must have enough capacity to allocate those VMs too. A standard OpenShift installation ends up with more that 50 Amphora Load Balancers.
```

Comment:
How big is an Amphora? I don't picture what are the HW requirements for 50 of them -- could be valuable information for anyone sizing their OpenStack cloud.

Reply:
amphora size is up to the admins, normal size is 1 vcpu and 1 GB of RAM. But it is up to the cloud admin to configure the amphora flavor. So they should know how much 50*amphora flavor is. And the tenant (the one installing OCP) only cares about the number of loadbalancer quota, not the size of the amphora